map/framework.cpp

```diff
@@ -2712,11 +2735,18 @@ osm::Editor::SaveResult Framework::SaveEditedMapObject(osm::EditableMapObject em
       shouldNotify = true;
     }
 
+    // TODO(mgsergio): Consider this:
+    // A user changed a feature with no original addres and addres differ from the one
```

Comment:
User has changed feature with no original address, and entered address is different from the one